<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>148</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>105</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QLaunchOverSSH</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/images/applet-gnome-sshmenu-icone-6595-16.png</normaloff>:/new/images/applet-gnome-sshmenu-icone-6595-16.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>35</y>
      <width>450</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_command">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>115</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Commande :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_command">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>6</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Sélectionnez la commande voulue</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="button_launch">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>454</width>
      <height>27</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Ok</set>
    </property>
    <property name="centerButtons">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="action_import"/>
    <addaction name="action_export"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_about">
   <property name="text">
    <string>À propos</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="action_import">
   <property name="text">
    <string>Importer...</string>
   </property>
  </action>
  <action name="action_export">
   <property name="text">
    <string>Exporter...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
